core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -602,11 +680,19 @@ export class LoggedInUser extends BaseUser {
       this.page.waitForNavigation(),
       this.clickOn(watchAVideoButton),
     ]);
-    if (this.page.url() !== watchAVideoUrl) {
+    const url = this.page.url().split('?')[0];
+    if (
```

Comment:
Better: define a variable `deviceHasMobileWidth` that stores whether `this.viewport.width < BREAKPOINTS.MOBILE`. Then define expectedUrl as `deviceHasMobileWidth ? ... : ...`. Then compare url to expectedUrl.